README.md

```diff
@@ -283,6 +283,14 @@ please consider using the method demonstrated in examples below, where the token
 **Supported GGUF tokenizer types**
 - `llama`
 
+Some GGUF models are very large and are sharded into multiple files. Mistral.rs supports this, and to use it, delimit the `.gguf` filenames with a space as such:
+
+```bash
+./mistralrs-server --chat-template <chat_template> gguf -m . -f "a.gguf b.gguf"
+```
+
+For the Python API, a list of strings is also accepted for this case.
```

Comment:
Do you have an example GGUF file where this is actually the case? Or was this a misunderstanding from referenced motivation in https://github.com/EricLBuehler/mistral.rs/issues/380 ?

Reply:
Yes: https://huggingface.co/mradermacher/dolphin-2.9-mixtral-8x22b-GGUF/tree/main

Reply:
I don't understand why the sharding is necessary other than as [this comment](https://www.reddit.com/r/LocalLLaMA/comments/1c2dfv6/comment/kzb1qpt/) suggests, to workaround size restrictions on a file host?

Seems like it'd be more appropriate to have a tool that concatenates the files together if there's a UX issue you want to address, but I don't really see the point in adding additional complexity / maintenance to support something that should realistically be addressed outside of `mistral.rs` (runtime) 🤷‍♂️ 

Similar to the tokenizer patching you added recently, these are workarounds that seem more appropriate as CLI tools to apply "fixes" 🤔